pages/common/phpstorm.md

```diff
@@ -0,0 +1,20 @@
+# phpstorm
+
+> A cross-platform IDE for PHP based on the JetBrains IntelliJ platform.
```

Comment:
Perhaps it would be a good idea to mention that it's commercial (i.e. paid-for) here? Or would we assume that the reader knows that if they are reading the page

Reply:
Free for open source. :+1: I guess maybe.

Reply:
Oh right! I hadn't heard about that. It must be new.

Reply:
I've been using it free for OS for 3 and a bit years now. 😋

Reply:
The commercial versions are free if you are student and have a university email address.

Reply:
They have a list of available offers. 👍
https://www.jetbrains.com/phpstorm/buy/#discounts